metrics/src/main/java/cucumber/metrics/regulator/module/SpeedRegulatorModule.java

```diff
@@ -0,0 +1,49 @@
+package cucumber.metrics.regulator.module;
```

Comment:
The point I'm trying to make is that the `metrics` sub-project is suppose to provide generic functionality to be able to meter the execution of the tests to get the time performance metrics.  The `SpeedRegulatorModule` class is using internally Google Guice specific functionality to inject the metrics components into the IOC.  Given the layout of the cucumber-jvm project, there are dedicated sub-projects to support integration of functions into various frameworks like Spring, Guice, Pico, etc, so IMHO the code that deals specifically with wiring things using Guice belongs in `guice` sub-project, to keep dependency separation clean.  This way `TimedInterceptor` can be used for all the different IOC containers such as Guice and Spring and Pico, etc to add AOP support for metric.